### DISCUSSION TITLE
Recommended way to generate API documentation

### DISCUSSION THREAD

**Discussion by Robstei** [Azure/azure-functions-nodejs-library#185] [Q&A]
Hey,

I was wondering if there is a recommeded / easy way to (auto)generate a documentation based on Typescript information?

I found [this](https://github.com/Azure/azure-functions-openapi-extension) for C# but nothing for nodejs/Typescript.

Regards

**Comment by ejizba** [comment]
That's cool I didn't know C# had that. Unfortunately no, I don't think there's anything like it for Node.js that's specific to Azure Functions. You might be able to hack something together with the generic (aka not specific to Azure) swagger tools that exist in the Node.js ecosystem - would be an interesting project for sure.

**Comment by Robstei** [comment]
Sry for the late response here and thank you for your fast response
